### PR TITLE
".w" whisper and different chat disable message

### PR DESCRIPTION
**Pull request by Brasnak**
Adds the ability to whisper by using ".w" as a prefix during Say. Also changes the chat disabled message from "Speech is currently admin-disabled" to "Speech has currently been disabled by the admins."

### CODE REVIEW

**Comment by SamHPurp** [comment]
Leading merge commits againnnnnnnn

File: code/modules/mob/say.dm [hunk 1]
```diff
@@ -2,17 +2,21 @@
 /mob/verb/say_verb(message as text)
 	set name = "Say"
 	set category = "IC"
+	var/regex/whisper_finder = regex("^\\.w\\s+|^\\.w")
 	if(GLOB.say_disabled)	//This is here to try to identify lag problems
-		to_chat(usr, "<span class='danger'>Speech is currently admin-disabled.</span>")
+		to_chat(usr, "<span class='danger'>Speech has currently been disabled by the admins.</span>")
```
Comment (FlattestGuitar) [inline code/modules/mob/say.dm]: This is actually grammatically incorrect, meine Freunde.

Comment (FlattestGuitar) [reply code/modules/mob/say.dm]: Or would it be "mein Freund"..?

I always get it mixed up.

**Comment by SamHPurp** [comment]
Can we be super fucking careful adding more regex.... They still haven't fixed the Regex that Flattest added to Paradise which stops the server from running nine times out of ten.

File: code/modules/mob/say.dm (hunk 1 above)
Comment (Brasnak) [reply code/modules/mob/say.dm]: "mein Freund" is singular, "meine Freunde" is plural.

**Comment by Brasnak** [comment]
i did test it, and so far, i have seen it cause no trouble at all

File: code/modules/mob/say.dm (hunk 1 above)
Comment (Brasnak) [reply code/modules/mob/say.dm]: Also, could you point out the grammatical error to me? I am failing to spot it.

**Comment by FlattestGuitar** [comment]
a) This regex is fine, it has no look-behind stuff, which apparently breaks BYOND on Linux (that's how I sabotaged Paradise back in the day)

b) `Speech is currently admin-disabled.` is correct. `Speech has recently been admin-disabled.` is also correct. `Speech has currently been admin-disabled.` is not correct. Leave it the way it was :P

**Comment by Brasnak** [comment]
Change reverted

**Pull request merged**